packages/yourloops/pages/hcp/team-members-table.tsx

```diff
@@ -315,30 +317,32 @@ function TeamMemberTable(props: TeamMembersProps): JSX.Element {
   const { t } = useTranslation("yourloops");
 
   return (
-    <Table id={`team-members-list-${team.id}-table`}>
-      {/* prettier-ignore */}
-      <TableHead className={classes.tableRowHeader}>
-        <TableRow>
-          <TableCell id={`team-members-list-${team.id}-cellheader-icon`} />
-          <TableCell id={`team-members-list-${team.id}-cellheader-lastname`}>
-            {t("lastname")}
-          </TableCell>
-          <TableCell id={`team-members-list-${team.id}-cellheader-firstname`}>
-            {t("firstname")}
-          </TableCell>
-          <TableCell id={`team-members-list-${team.id}-cellheader-email`}>
-            {t("email")}
-          </TableCell>
-          <TableCell id={`team-members-list-${team.id}-cellheader-role`}>
-            {t("team-member-admin")}
-          </TableCell>
-          <TableCell id={`team-members-list-${team.id}-cellheader-actions`} />
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        <MembersTableBody {...props} classes={classes} />
-      </TableBody>
-    </Table>
+    <TableContainer component={Paper}>
```

Comment:
Why adding an upper component here ?

Reply:
I think all tables should have the same design for consistency, like in patients page. Mathieu approved these changes